tests/node/test_urllib3_chain_certs.py

```diff
@@ -96,4 +90,4 @@ def test_assert_fingerprint_in_cert_chain_failure(node_cls):
         in err
     )
     # This is the root CA for httpbin.org with a leading comma to denote more than one cert was listed.
-    assert ', "8ecde6884f3d87b1125ba31ac3fcb13d7016de7f57cc904fe1cb97c6ae98196e"' in err
+    assert ', "cbb522d7b7f127ad6a0113865bdf1cd4102e7d0759af635a7cf4720dc963c53b"' in err
```

Comment:
The certificate is currently generated by trustme: https://github.com/kevin1024/pytest-httpbin/pull/90. Since it generates its own CA, this is going to be less stable than before. Additionally, the comment is out of date.

Reply:
This is the root CA for www.elastic.co, this is the one test that does not use the local httpbin.